### ISSUE TITLE
Tests failing because of timeout

### ISSUE THREAD

**Issue by gravesm** [ansible-collections/kubernetes.core#280]
This issue will be used to track tests that fail due to timeouts. This is a recurring problem with our CI. If you have a build that fails because of a timeout, link the test here. **Please don't close this issue**.

- [ ] https://github.com/ansible-collections/kubernetes.core/blob/bf26f5a3be025822f8f26250c09f39d8ff73da11/molecule/default/roles/helm/tasks/tests_chart.yml#L290

**Comment by goneri** [comment]
I've increased the timeout to 2hours. Ideally, we should split these jobs.

**Comment by gravesm** [comment]
Yeah. If we're going to switch to ansible-test, we should do that first, though. I think splitting the jobs will look very different for molecule as opposed to ansible-test.

**Comment by gravesm** [comment]
I'm going to close this as it has become less of a problem with the switch the ansible-test and the use of the test splitter.
